automation/test.sh

```diff
@@ -74,5 +81,27 @@ done
 kubectl get pods -n ${NAMESPACE}
 kubectl version
 
+# Prepare PV for windows testing
+if [ -n "$NFS_WINDOWS_DIR" ]; then
```

Comment:
If we check here for the directory existence too, instead of the variable content, we can make the location easily configruable.

Reply:
fixed